share/harvesters/com_mendeley_data.py

```diff
@@ -0,0 +1,82 @@
+from furl import furl
+import base64
+import pendulum
+
+from django.conf import settings
+
+from share.harvest import BaseHarvester
+
+
+class MendeleyHarvester(BaseHarvester):
+    VERSION = 1
+
+    def get_token(self):
+        """ Mendeley gives tokens that last for one hour. A new token will be
+        requested everytime the harvester is run to ensure the access token is
+        valid.
+        """
+        url = 'https://api.mendeley.com/oauth/token'
+        username_password = settings.MENDELEY_API_CLIENT_ID + ':' + settings.MENDELEY_API_CLIENT_SECRET
```

Comment:
I'm not following. You want the client ID and the client secret to be harvester kwargs and not environment variables?